### PR TITLE
Add initial helm chart for graylog

### PR DESCRIPTION
**Pull request by bjaworski3**
**What this PR does / why we need it**: Adds Helm chart for graylog

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by alexander-alvarez** [comment]
@edmundoa can someone in your org take a look at this?

**Comment by bernd** [comment]
@alexander-alvarez I can take a look over the next days.

**Comment by unguiculus** [comment]
Please compare with what you get from `helm create` using Helm 2.8+ and update accordingly.

**Comment by bjaworski3** [comment]
@unguiculus I did my tests with the chart on 2.9.1, did you want me to test it pre 2.8?

**Comment by davidkarlsen** [comment]
@bjaworski3 Still want this merged? I suggest going straight for stable and not via incubating

**Comment by bjaworski3** [comment]
@davidkarlsen Yeah it would still be good to get this merged. I can move it over to the stable folder

File: stable/graylog/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+* Create a dynamically scalable cluster of worker nodes
+
+### Installing the Chart
+
+To install the chart with the release name `my-graylog` in the default
+namespace:
+
+```
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
```
Comment (davidkarlsen) [inline stable/graylog/README.md]: still references incubator - also in other places

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/graylog/README.md (hunk 1 above)
Comment (davidkarlsen) [reply stable/graylog/README.md]: also see linting errors for whitespaces

**Comment by billimek** [comment]
@bjaworski3 did you give up on this one?  If so I'll pick it up as I'd really like to see a graylog chart.

**Comment by billimek** [comment]
@bjaworski3 good question - I know some PRs take a long time to get movement, let's try to get this one across the finish-line!  Can you post a comment with `/reopen` ?  I think that should instruct the bot to reopen it - I'll try but I think that author needs to do it.

/reopen

**Comment by billimek** [comment]
@davidkarlsen or @unguiculus or @bernd is it possible to get this PR `/reopen`ed to (hopefully) complete the path to getting this merged?

**Comment by billimek** [comment]
@bjaworski3 any chance you can submit this as a new PR?  I don't think we're having any luck getting anyone to reopen this.  I asked in the #charts channel and the response was to create a new PR instead of re-opening this one.  It doesn't make a lot of sense to me in that all of the history will be split across multiple PRs but maybe that's the only way to get some progress on this.

**Pull request closed without merging**